scripts/config.py

```diff
@@ -59,23 +59,28 @@ def __init__(self):
 
         self.use_mac_os_tts = False
         self.use_mac_os_tts = os.getenv("USE_MAC_OS_TTS")
-        
```

Comment:
Rollback the whitespace fix. It's painful but it reduces the conflicts. The whitespace PR is already there.

Reply:
rollback down